.github/workflows/stale.yml

```diff
@@ -1,7 +1,7 @@
-name: 'Close Stale Issues and PRs'
+name: 'Close stale issues and PRs'
 on:
   schedule:
-    - cron: '0 * * * *'
+    - cron: '0 * * * *' # every hour
```

Comment:
Do we need this to run every hour? I think just once or twice a day should be sufficient?